packages/squiggle-lang/__tests__/Symbolic_test.res

```diff
@@ -0,0 +1,30 @@
+open Jest
+open Expect
+open Js.Array
+open SymbolicDist
+
+let makeTest = (~only=false, str, item1, item2) =>
+  only
+    ? Only.test(str, () => expect(item1) -> toEqual(item2))
+    : test(str, () => expect(item1) -> toEqual(item2))
+
+let normalParams1: SymbolicDistTypes.normal = {mean: 5.0, stdev: 2.0}
```

Comment:
why is this called ``normalParams1``?

Reply:
This is in line with how I've always named unit test cases in every codebase I've been a part of. I can definitely switch it over to `normalDistParams1`, or `normalDistAtMean5`. 

I've never really thought of unit test input naming as an important problem to think about.

Reply:
I like ``normalDistAtMean5``. 

I think that getting naming right doesn't take all too much time. Having readable tests seems pretty useful to me, though not as useful as having readable regular code. (It's also just good practice)

Reply:
stuff like `normalDistAtMean5` and `pdfNormalDistAtMean15` is in the next commit.